package-lock.json

```diff
@@ -799,6 +799,21 @@
                 "ms-rest-azure": "^2.5.5"
             }
         },
+        "azureintegration_repoanalysis_client_internal": {
```

Comment:
do we have `_` in package name? I think it should be `-`. I am not really sure if this could be a problem as I have not read so anywhere but lets just follow the common practice.